### DISCUSSION TITLE
Notification when switching in Airplane mode

### DISCUSSION THREAD

**Discussion by cavemaster** [seemoo-lab/AirGuard-iOS#2] [General]
Hi,

for the night I'm activating the airplane mode.
Unfortunately after a couple of minutes I get an internal notification that AirGuard don't work probalby in this mode.
Is there a way how I can deactivate this notification when I'm trying to sleep? :-)

**Comment by Sn0wfreezeDev** [comment]
Hi, 
currently we don't have an option for that. Does it alarm even when using the sleep mode on iOS. 
We can add a switch to disable these notifications in the next update.

**Reply by cavemaster** [reply]
Yes it does but the notification disappears automatically so I can't see it in the history.
Would be great if this could be a feature for the next update.
Great software!

**Reply by leonboe1** [reply]
Hi, you can go to iOS settings → Notifications → AirGuard and disable "Sound". This disables the sound and the vibration if there is a notification sent. Then, you shouldn't be disturbed anymore.
